MAUI/DateTimePicker/date-time-picker-mode.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Date Time picker mode in .NET MAUI Date Time Picker control | Syncfusion
```

Comment:
title: Date Time picker mode in .NET MAUI Date Time Picker Control | Syncfusion

Reply:
Content modified.